app/components/auk/tab.js

```diff
@@ -41,4 +41,12 @@ export default class Tab extends Component {
     }
     return null;
   }
+
+  get query() {
+    if (isPresent(this.args.query)) {
+      return this.args.query;
+    }
+    // return empty object
+    return [];
+  }
```

Comment:
The reason for this and removal of `@query` elsewhere is that with recent dependency version, passing this as undefined throws errors.
Also `@query` is only used when used in `<auk::tab>`